src/main/java/utility/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package utility;
+
+import java.util.Random;
+import model.Color;
+
+public class ColorSupplier {
+    private static final Random random = new Random();
+    private static final Color[] colors = Color.values();
+
+    public String getRandomColor() {
+        final int index = random.nextInt(colors.length);
+        return colors[index].name().toLowerCase();
+    }
```

Comment:
The method `getRandomColor` returns a color in lowercase. It's better to return the enum constant's name as it is, without changing its case, to maintain consistency. If the case transformation is needed, it should be done outside this utility method.